src/components/userProfileLayout/categoryDelete/categoryDelete.module.scss

```diff
@@ -61,6 +61,9 @@
 		& {
 			flex-direction: column;
 			gap: 10px;
+			> * {
+				width: 100%;
+			}
```

Comment:
```
gap: 10px;

button {
```

can you move this selector alter seletor's body leaving space beahind them here and in other similar places?